.github/workflows/rust_checks.yml

```
@@ -0,0 +1,125 @@
+name: Rust Static checks
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/rust_checks.yml
+      - Cargo.lock
+      - Cargo.toml
+      - charts/linkerd-control-plane/templates/destination-rbac.yaml
+      - charts/linkerd-crds/templates/policy/**
+      - deny.toml
+      - policy-controller/**
+      - policy-test/**
```

And then we can be more flexible here

```suggestion
      - '**/*.rs'
      - policy-*/*.dockerfile
```